__tests__/user/user.query.test.js

```diff
@@ -33,7 +33,7 @@ describe('queries', () => {
 
         userId = register.data.registerUser._id
 
-        const auth = await client.mutate({
+        const authRes  = await client.mutate({
```

Comment:
Please, query all possible fields for the login mutation and check their values and types as much as possible

Reply:
fixed